core/src/main/java/jenkins/util/groovy/GroovyHookScript.java

```diff
@@ -79,10 +83,10 @@ public void run() {
             }
         }
 
-        File script = new File(j.getRootDir(), hookGroovy);
+        File script = new File(home, hookGroovy);
```

Comment:
Never mind. I missed the null check in [BootFailure](https://github.com/jglick/jenkins/blob/WebAppMain.contextDestroyed-logging/core/src/main/java/hudson/util/BootFailure.java#L43).